pkg/render/passthru.go

```diff
@@ -2,18 +2,25 @@ package render
 
 import (
 	operatorv1 "github.com/tigera/operator/api/v1"
+	"github.com/tigera/operator/pkg/common"
 	rmeta "github.com/tigera/operator/pkg/render/common/meta"
+	"github.com/tigera/operator/pkg/tls/certificatemanagement"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 func NewPassthrough(objs ...client.Object) Component {
 	return &passthroughComponent{objs: objs}
 }
 
+func NewDeletionPassthrough(objsToDelete ...client.Object) Component {
```

Comment:
I don't think we should have a separate function for this, it seems overkill. If we need this, then I think we should either:
1. Modify the `NewPassthrough` to accept objects to delete.
2. Export the PassThroughComponent and let the user directly set the objects to create / delete.

TBH I don't think there's a good reason to have a constructor for this, it's a prime candidate for a struct that's directly interacted with.

Reply:
I meant to delete. Lets get rid of it.